Corona-Warn-App/src/test/java/de/rki/coronawarnapp/appconfig/AppConfigProviderTest.kt

```diff
@@ -55,170 +56,154 @@ class AppConfigProviderTest : BaseIOTest() {
-        runBlocking {
-            createDownloadServer().getAppConfig().minRiskScore shouldBe 11
-        }
+        createDownloadServer().getAppConfig().minRiskScore shouldBe 11
     }
 
     @Test
-    fun `fallback to last config if download fails`() {
+    suspend fun `fallback to last config if download fails`() {
         mockConfigStorage = APPCONFIG_RAW.toByteArray()
 
         coEvery { api.getApplicationConfiguration("DE") } throws Exception()
 
         every { verificationKeys.hasInvalidSignature(any(), any()) } returns false
 
-        runBlocking {
-            createDownloadServer().getAppConfig().minRiskScore shouldBe 11
-        }
+        createDownloadServer().getAppConfig().minRiskScore shouldBe 11
     }
 
     // Because the UI requires this to detect when to show alternative UI elements
     @Test
-    fun `if supportedCountryList is empty, we do not insert DE as fallback`() {
+    suspend fun `if supportedCountryList is empty, we do not insert DE as fallback`() {
         coEvery { api.getApplicationConfiguration("DE") } returns APPCONFIG_BUNDLE.toResponseBody()
         every { verificationKeys.hasInvalidSignature(any(), any()) } returns false
 
-        runBlocking {
-            createDownloadServer().getAppConfig().supportedCountriesList shouldBe emptyList()
-        }
+        createDownloadServer().getAppConfig().supportedCountriesList shouldBe emptyList()
     }
 
     companion object {
         private val APPCONFIG_BUNDLE =
-            ("504b0304140008080800856b22510000000000000000000000000a0000006578706f72742e62696ee3e016" +
-                "f2e552e662f6f10f97e05792ca28292928b6d2d72f2f2fd74bce2fcacf4b2c4f2ccad34b2c28e0" +
-                "52e362f1f074f710e097f0c0a74e2a854b80835180498259814583d580cd82dd814390010c3c1d" +
-                "a4b8141835180d182d181d181561825a021cac02ac12ac0aac40f5ac16ac0eac86102913072b3e" +
-                "01460946841e47981e25192e160e73017b21214e88d0077ba8250fec1524b5a4b8b8b858043824" +
-                "98849804588578806a19255884c02400504b0708df2c788daf000000f1000000504b0304140008" +
-                "080800856b22510000000000000000000000000a0000006578706f72742e736967018a0075ff0a" +
-                "87010a380a1864652e726b692e636f726f6e617761726e6170702d6465761a0276312203323632" +
-                "2a13312e322e3834302e31303034352e342e332e321001180122473045022100cf32ff24ea18a1" +
-                "ffcc7ff4c9fe8d1808cecbc5a37e3e1d4c9ce682120450958c022064bf124b6973a9b510a43d47" +
-                "9ff93e0ef97a5b893c7af4abc4a8d399969cd8a0504b070813c517c68f0000008a000000504b01" +
-                "021400140008080800856b2251df2c788daf000000f10000000a00000000000000000000000000" +
-                "000000006578706f72742e62696e504b01021400140008080800856b225113c517c68f0000008a" +
-                "0000000a00000000000000000000000000e70000006578706f72742e736967504b050600000000" +
-                "0200020070000000ae0100000000").decodeHex()
+            (
```

Comment:
Try to not include unnecessary reformatting.

Reply:
Ok, sorry for that. I did a command+option+L on the file and it must have changed this as well. I'll try to not include these kinds of changes in the future.